app/services/case_filter/open_case_status_filter.rb

```diff
@@ -26,11 +26,21 @@ def available_choices
           end
         end
       end
+
+      # To ensure 'rejected' is at the end of the filter list, .delete returns the
```

Comment:
The explanation for what is being done should probably be with the change_case_filter_order method. Its name could maybe be changed to make it clear that the element is being moved to the end of the hash?